### PR TITLE
feat(interactions tab dates grid): created the interactions tabs dates grid

### PR DESCRIPTION
**Pull request by MayaSasson**

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionsTab.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { format } from 'date-fns';
+
+import useContent from './useInteractionsTab';
+import { StartInvestigationDateVariablesConsumer } from '../../StartInvestigationDateVariables/StartInvestigationDateVariables';
+import { Card, Typography, Button } from '@material-ui/core';
```
Comment (roymaniv) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionsTab.tsx:6]: imports

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionsTab.test.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                }
+                const recievedDates = useInteractionsTabOutcome.getDatesToInvestigate(startInvestigationDateVariables);
+                
+                expect(recievedDates).toEqual(eachDayOfInterval({start: subDays(symptomsStartDate, 4), end: endInvestigationDate}));
+            })
+
+            it('get dates when both symptoms start and exposure date are available', async () => {
+                const endInvestigationDate = new Date();
+                const symptomsStartDate: Date = subDays(endInvestigationDate, 2);
+                const exposureDate = subDays(new Date(), 5);
```
Comment (RoeiRom) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionsTab.test.tsx:37]: use end investigation date

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionsTab.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import { format } from 'date-fns';
+import { Card, Typography, Button } from '@material-ui/core';
+
+import useContent from './useInteractionsTab';
+import useStyles from './InteractionsTabStyles';
+import { StartInvestigationDateVariablesConsumer } from '../../StartInvestigationDateVariables/StartInvestigationDateVariables';
+
+const InteractionsTab: React.FC = (): JSX.Element => {
+    const classes = useStyles({});
+    const { getDatesToInvestigate } = useContent();
```
Comment (RoeiRom) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionsTab.tsx:11]: change to useInteractionsTab

File: client/src/components/App/Content/InvestigationForm/TabManagement/TabManagement.tsx [hunk 4]
```diff
@@ -2,7 +2,10 @@ import React from 'react';
 import {Paper, Tabs, Tab, Card, createStyles, withStyles} from '@material-ui/core';
 
 import { Tab as TabObj } from 'models/Tab';
+
 import useStyles from './TabManagementStyles';
+import ClinicalDetails from './ClinicalDetails/ClinicalDetails';
```
Comment (RoeiRom) [inline client/src/components/App/Content/InvestigationForm/TabManagement/TabManagement.tsx]: remove

File: client/src/components/App/Content/InvestigationForm/TabManagement/TabManagement.tsx [hunk 5]
```diff
@@ -15,19 +18,22 @@ export const tabs: TabObj[] = [
     defaultTab,
     {
         id: 1,
-        name: 'בידוד ופרטים קליניים',
+        name: 'בידוד ופרטים קליניים', 
+        displayComponent: <ClinicalDetails />,
```
Comment (RoeiRom) [inline client/src/components/App/Content/InvestigationForm/TabManagement/TabManagement.tsx]: change to <></>

File: client/src/models/TabObj.ts [hunk 6]
```diff
@@ -0,0 +1,7 @@
+interface TabObj {
```
Comment (RoeiRom) [inline client/src/models/TabObj.ts:1]: remove file

**Review by RoeiRom — CHANGES_REQUESTED** [review]

**Review by roymaniv — APPROVED** [review]

**Review by RoeiRom — APPROVED** [review]

**Pull request merged**